src/services/branch-manager.js

```diff
@@ -1,3 +1,13 @@
+const P = require('bluebird');
+const agent = require('superagent-promise')(require('superagent'), P);
+const authenticator = require('./authenticator');
+const { serverHost } = require('../config');
+
+const ERROR_MESSAGE_PROJECT_IN_V1 = '⚠️  This project does not support branches yet. Please migrate your environments from your Project settings first.';
+const ERROR_MESSAGE_ENV_SECRET_ISSUE = '⚠️  Your development environment is not properly set up. Please run `forest init` first and retry.';
+const ERROR_MESSAGE_BRANCH_ALREADY_EXISTS = '❌ This branch already exists.';
+const ERROR_MESSAGE_NO_PRODUCTION_OR_REMOTE_ENVIRONMENT = '❌ You cannot create a branch until this project has either a remote or a production environment.';
```

Comment:
I think this error might also happend for all the commandes, so maybe it should be rephrased (Like `You cannot run branch commands until this project has either a remote or a production environment.`) ?